source/extensions/filters/network/ext_authz/ext_authz.cc

```diff
@@ -28,7 +28,8 @@ void Filter::callCheck() {
   config_->stats().total_.inc();
 
   calling_check_ = true;
-  client_->check(*this, check_request_, Tracing::NullSpan::instance());
+  client_->check(*this, check_request_, Tracing::NullSpan::instance(),
+                 filter_callbacks_->connection().streamInfo());
```

Comment:
i assume this means that there is limited support for this /w the network filter as well (e.g. to embed the downstream SAN or something). Maybe add a test for that since I'm sure people will start relying on it?

Reply:
Done.